src/main/python/rlbot/setup_manager.py

```diff
@@ -11,6 +11,8 @@
 from pathlib import Path
 from typing import List, Optional, Dict
 from urllib.parse import ParseResult as URL
+import configparser
+import atexit
```

Comment:
Do these require special packages to be installed from pypi, or are they built in to python? If it's the former, then there's some additional work to be done in setup.py.

Reply:
These are built in packages :]